src/Dev/Tasks/FileMigrationHelper.php

```diff
@@ -317,10 +310,10 @@ protected function migrateFile($base, File $file, $legacyFilename)
             return false;
         }
 
-        $this->logger->info(sprintf('* SS3 file %s converted to SS4 format', $file->getFilename()));
+        $this->logger->debug($file->getFilename());
```

Comment:
Is this supposed to be on the same line as the next debug messages?

Reply:
No, it's the "section headline" so to speak - each file can have one or more thumbnails related to it, which are their own indented log messages.